common/eth2/src/lib.rs

```diff
@@ -206,12 +206,14 @@ impl BeaconNodeHttpClient {
         self.get_opt(path).await
     }
 
-    /// `GET beacon/states/{state_id}/validators`
+    /// `GET beacon/states/{state_id}/validators?id,status`
     ///
     /// Returns `Ok(None)` on a 404 error.
     pub async fn get_beacon_states_validators(
         &self,
         state_id: StateId,
+        ids: Option<&[ValidatorId]>,
+        statuses: Option<&[ValidatorStatus]>,
```

Comment:
I see that giving `Some(&[])` would yield no results. This seemed odd at first, but after giving it some thought I think it's my preference.

No change requested :)